docs/source/miscellaneous/stubs.rst

```diff
@@ -539,46 +689,57 @@ the 3-parameter signatures:
+option.
+
+Running ``./build/gen_stub.php --generate-methodsynopses ./ext/mbstring
+../doc-en/reference/mbstring`` will create a dedicated page for each ``ext/mbstring`` function which
+is not yet documented, and saves them into the ``../doc-en/reference/mbstring/functions`` directory.
 
-First of all, newly added functions or methods can be documented by providing the
-``--generate-methodsynopses`` option. E.g. running ``./build/gen_stub.php --generate-methodsynopses
-./ext/mbstring ../doc-en/reference/mbstring`` will create a dedicated page for each ``ext/mbstring``
-function which is not yet documented, saving them into the
-``../doc-en/reference/mbstring/functions`` directory. Since these are stub pages, many of the
-sections are empty by default, so the relevant descriptions have to be added, while the irrelevant
-ones have to be removed.
+Since these are stub documentation pages, many of the sections are empty. Relevant descriptions have
+to be added, and irrelevant sections should be removed.
 
-For functions or methods which are already available in the manual, the documented signatures can be
-updated by providing the ``--replace-methodsynopses`` option. E.g. running ``./build/gen_stub.php
---replace-methodsynopses ./ ../doc-en/`` will update all the function or method signatures in the
-English documentation whose stub counterpart is found.
+Functions or methods that are already available in the manual, the documented signatures can be
+updated by providing the ``--replace-methodsynopses`` option.
+
+Running ``./build/gen_stub.php --replace-methodsynopses ./ ../doc-en/`` will update the function or
+method signatures in the English documentation whose stub counterpart is found.
 
 Class signatures can be updated in the manual by providing the ``--replace-classsynopses`` option.
-E.g. running ``./build/gen_stub.php --replace-classsynopses ./ ../doc-en/`` will update all the
-class signatures in the English documentation whose stub counterpart is found.
+
+Running ``./build/gen_stub.php --replace-classsynopses ./ ../doc-en/`` will update all the class
+signatures in the English documentation whose stub counterpart is found.
 
 If a symbol is not intended to be documented, the ``@undocumentable`` PHPDoc tag should be added to
-it. Doing so will prevent any documentation to be created for the given symbol. In order not to add
-a whole stub file to the manual, the PHPDoc tag should be applied to the file itself. These
-possibilities are useful for symbols which exist only for testing purposes (e.g. the ones declared
-for ``ext/zend_test``), or by some other reason documentation is not possible.
+it. Doing so will prevent any documentation to be created for the given symbol.
+
+To avoid a whole stub file to be added to the manual, this PHPDoc tag should be applied to the file
+itself.
+
+These flags are useful for symbols which exist only for testing purposes (e.g. the ones declared for
```

Comment:
This sentence is related to the previous one so it should go into the same paragraph, shouldn't it?

Reply:
Will fix.